install.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+if [ $# -lt 1 ]; then
+  echo "Error: missing arguments. Usage: $0 [linux|cobc] [prefix]"
+  exit 1
+fi
+
+if [[ $1 != "linux" && $1 != "cobc" ]]; then
+  echo "Error: first argument must be either 'linux' or 'cobc'"
+  exit 1
+fi
+
+if [[ $1 == "cobc" && $# -lt 2 ]]; then
+  echo "Error: missing prefix argument for 'cobc' option"
+  exit 1
+fi
+
+echo "Arguments provided: $1 $2"
+
+if [ -f /.dockerenv ]; then
+    DOCKER_BUILD=true
+else
+    DOCKER_BUILD=false
+fi
+
+PARAM_FILE="libraries.txt"
+
+while IFS=, read -r name commit repo_url; do
+  echo "Cloning $name at commit $commit from $repo_url"
+  git clone "$repo_url"
+  cd "$name"
+  git checkout -q "$commit"
+  cd ..
+done < "$PARAM_FILE"
+
+
+cd rodos
+# We will need it later, so just copy it to the top-level directory
+find . -name linux-x86.cmake | xargs cp -t ../ -v
+if [[ $1 == "linux" ]]; then
+  cmake --toolchain cmake/port/linux-x86.cmake -S . -B build
+  cmake --build build
+  sudo cmake --install build
```

Comment:
Use `build/linux-x86` as build directory for the Linux version of Rodos to make it coherent with the build directory for the COBC version.